common/plugins/eu.esdihumboldt.hale.common.headless/src/eu/esdihumboldt/hale/common/headless/transform/filter/InstanceFilterDefinition.java

```diff
@@ -183,20 +186,25 @@ private boolean applyTypedFilter(Instance instance) {
 				return false;
 			}
 		}
-		// it reaches here that means Instance does not match any type of type
-		// filters.
-		// Method should return true.
+		// it reaches here that means Instance does not match any type of
+		// type filters. Method should return true.
 		return true;
```

Comment:
But returning true here will result in the OR to yield always true, meaning that when an instance's type does not match the type of a type filter it will always be included, no matter what the result of the unconditional filters would be.